ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3352,3 +3352,30 @@ literals:
 - low_heating_supply_water_temperature_alarm:
   - ACTIVE
   - INACTIVE
+- isolation_damper_percentage_command:
+    fixed_min: 0.0
+    fixed_max: 100.0
+- failed_isolation_damper_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_zone_air_refrigerant_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_zone_air_co_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_zone_air_relative_humidity_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_supply_air_damper_alarm:
+  - ACTIVE
+  - INACTIVE
+- plenum_static_pressure_sensor:
+    flexible_min: -249.080024
+    flexible_max: 249.080024
+- low_limit_plenum_static_pressure_setpoint:
+    flexible_min: -249.080024
+    flexible_max: 249.080024
+- low_plenum_static_pressure_alarm:
```

Comment:
this is incorrect. you need to specify that it is air_static pressure

additonally, no need to define plenum -- `low_supply_air_static_pressure_alarm` should suffice